content/html/concepts/tags/terms/audio/audio.md

```diff
@@ -46,4 +46,9 @@ The `<audio>` tag may include `<source>` tags and fallback text. It may also use
 </audio>
 ```
 
+<audio controls>
+  <source src="https://raw.githubusercontent.com/Codecademy/docs/main/media/introducing_docs_audio.mp3" type="audio/mp3" />
+  <h3>Fallback text if audio will not play</h3>
```

Comment:
When viewing the file I noticed this displays as a header on the page. Is this what we are going for?

Reply:
@KTom101 A guess that GH MD doesent support the audio tag so it goes back to the fallback audio.

Reply:
@ethan-master-coding I think we should just go with plain text and remove the `<h3>`s for both the `<video>` and `<audio>` example outputs.

Reply:
```
<audio controls>
  <source src="https://raw.githubusercontent.com/Codecademy/docs/main/media/introducing_docs_audio.mp3" type="audio/mp3" />
  Fallback text if audio will not play
</audio>

Reply:
@Dusch4593 I made that change.